privaterelay/management/commands/invite_monitor_waitlist.py

```diff
@@ -1,30 +1,88 @@
 from datetime import datetime
+from email.utils import parseaddr
+import logging
 
+from socketlabs.injectionapi.message.basicmessage import BasicMessage
+from socketlabs.injectionapi.message.emailaddress import EmailAddress
+
+from django.conf import settings
 from django.core.management.base import BaseCommand
+from django.template.loader import render_to_string
 
+from emails.utils import get_socketlabs_client, socketlabs_send
 from ...models import Invitations, MonitorSubscriber
 
 
+logger = logging.getLogger('events')
+
+
 class Command(BaseCommand):
     help = 'Connects to Monitor waitlist and creates invitations.'
 
+    def add_arguments(self, parser):
+        parser.add_argument('limit', nargs=1, type=int)
+
     def handle(self, *args, **options):
+        limit = options['limit'][0]
+
         monitor_waitlist = MonitorSubscriber.objects.using('monitor').filter(
             waitlists_joined__email_relay__notified=False
-        )
+        )[:limit]
+
         print("adding %s invitations ..." % monitor_waitlist.count())
+
         for invitee in monitor_waitlist:
             try:
                 Invitations.objects.get(
                     email=invitee.primary_email,
                     active=True,
                 )
-                print("%s already has an active invitation" % invitee.primary_email)
+                print(
+                    "%s already has an active invitation" %
+                    invitee.primary_email
+                )
```

Comment:
@groovecoder is there any plan to aggregate the "already active invitation" data and update the `MonitorSubscriber` table so that `waitlists_joined__email_relay__notified` is `True` if not I recommend updating the `invitee` here:
```suggestion
                )
                invitee.waitlists_joined['email_relay']['notified'] = True
                invitee.save(update_fields=['waitlists_joined'])
```

Reply:
Hmm ... good question. I'm trying to think if there's a situation where someone would have an active `Invitation` in Relay, but wouldn't be notified. I don't think there is, so yeah - this could set `notified` to `True` if it finds one.